packages/styles/src/variables/_spacing.scss

```diff
@@ -1,181 +1,57 @@
+  1: $size-1,
+  2: $size-2,
+  4: $size-4,
+  8: $size-8,
+  16: $size-16,
+  24: $size-24,
+  32: $size-32,
+  40: $size-40,
+  48: $size-48,
+  64: $size-64,
+  80: $size-80,
+  96: $size-96,
+  112: $size-112,
 );
 
-// This variable affects the `.h-*` and `.w-*` classes.
-$sizes: () !default;
-$sizes: map-merge(
+// create auto-spacers
+$auto-spacers: map-merge(
+  $spacers,
   (
-    25: 25%,
-    50: 50%,
-    75: 75%,
-    100: 100%,
     auto: auto,
-  ),
-  $sizes
+  )
 );
 
-// Abstand Gross, Section
-$size-curve-giant: (
-  'xs': $size-huge,
-  'sm': $size-huge,
-  'rg': $size-huge,
-  'md': $size-giant,
-  'lg': $size-giant,
-  'xl': $size-giant,
-  'xxl': $size-giant,
-) !default;
-
-// Abstand standard
-$size-curve-huge: (
-  'xs': $size-big,
-  'sm': $size-big,
-  'rg': $size-big,
-  'md': $size-bigger-big,
-  'lg': $size-bigger-big,
-  'xl': $size-huge,
-  'xxl': $size-huge,
-) !default;
-
-// Abstand Section halb
-$size-curve-bigger-big: (
-  'xs': $size-large,
-  'sm': $size-large,
-  'rg': $size-large,
-  'md': $size-small-huge,
-  'lg': $size-small-huge,
-  'xl': $size-small-huge,
-  'xxl': $size-small-huge,
-) !default;
-
-// Abstand Breadcrumb, Stepper
-$size-curve-big: (
-  'xs': $size-small-large,
-  'sm': $size-small-large,
-  'rg': $size-small-large,
-  'md': $size-big,
-  'lg': $size-big,
-  'xl': $size-bigger-big,
-  'xxl': $size-bigger-big,
-) !default;
-
-// H2
-$size-curve-large: (
-  'xs': $size-regular,
-  'sm': $size-regular,
-  'rg': $size-regular,
-  'md': $size-small-large,
-  'lg': $size-small-large,
-  'xl': $size-big,
-  'xxl': $size-big,
-) !default;
-
-// Button
-$size-curve-regular: (
-  'xs': $size-regular,
-  'sm': $size-regular,
-  'rg': $size-regular,
-  'md': $size-large,
-  'lg': $size-large,
-  'xl': $size-large,
-  'xxl': $size-large,
-) !default;
-
-// Teaser
-$size-curve-small: (
-  'xs': $size-small-regular,
-  'sm': $size-regular,
-  'rg': $size-regular,
-  'md': $size-regular,
-  'lg': $size-regular,
-  'xl': $size-regular,
-  'xxl': $size-regular,
-) !default;
-
-// Eingabefelder, p, h3, h4
-$size-curve-tiny: (
-  'xs': $size-small-regular,
-  'sm': $size-small-regular,
-  'rg': $size-small-regular,
-  'md': $size-regular,
-  'lg': $size-regular,
-  'xl': $size-regular,
-  'xxl': $size-regular,
-) !default;
-
-// List of all adaptive size curves
-$size-curves: (
-  'giant': $size-curve-giant,
-  'huge': $size-curve-huge,
-  'bigger-big': $size-curve-bigger-big,
-  'big': $size-curve-big,
-  'large': $size-curve-large,
-  'regular': $size-curve-regular,
-  'small': $size-curve-small,
-  'tiny': $size-curve-tiny,
-) !default;
-
-$responsive-spacing-properties: (
-  'm': 'margin',
-  'mt': 'margin-top',
-  'mr': 'margin-right',
-  'mb': 'margin-bottom',
-  'ml': 'margin-left',
-  'p': 'padding',
-  'pt': 'padding-top',
-  'pr': 'padding-right',
-  'pb': 'padding-bottom',
-  'pl': 'padding-left',
-  'gap': 'gap',
-);
+// override bootstrap negative-spacers
+$negative-spacers: ();
```

Comment:
The `$negative-spacers` map is automatically generated from `$spacers` in `bootstrap/scss/maps`